### ISSUE TITLE
Tutorial videos do not show up on linux or steamdeck

### ISSUE THREAD

**Issue by kayila** [pardeike/Achtung2#77]
After installing this mod, when the tutorial window appears, most of the tabs are empty on linux and steamdeck. It looks like there is supposed to be a video or some embed, but there is only blank pages.

**Comment by pardeike** [comment]
You are missing a video codec. See if you can play these videos: https://github.com/pardeike/Achtung2/blob/master/Features/02_Forcing.mp4

**Comment by kayila** [comment]
Yes, I can watch that video just fine without any problem, however it does not appear when I'm playing Rimworld.

**Comment by pardeike** [comment]
Not sure what to do. In my Ubuntu test environment they play ok in the mod.
